client/services/distanceCalc.tsx

```diff
@@ -0,0 +1,9 @@
+import haversine from 'haversine-distance';
+
+const distToBike = (userLocation, bikeLocation) => {
+    return Math.round(
```

Comment:
Maybe round to one or two significant digit(s). So 5.1 miles or 5.12 miles instead of 5 miles.

Reply:
That returns distance in meters so I figured rounding to the nearest integer would suffice. I'll add some comments to make that clearer!